bt_exploration_node:
  ros__parameters:
    use_sim_time: True
    # config: ["48.0", "60.0", "48.0", "1.25", "1.25", "1.25", "1.25", "60.0"]
    config: ["310.0", "260.0", "310.0", "-120.0", "-70.0", "-120.0", "-70.0", "260.0"]
    bt_xml_path: "/root/dev_ws/src/roadmap_explorer/roadmap_explorer/xml/exploration_with_fi.xml"

explore_costmap:
  explore_costmap:
    ros__parameters:
      update_frequency: 5.0
      publish_frequency: 5.0
      global_frame: map
      robot_base_frame: base_footprint
      use_sim_time: True
      resolution: 0.05
      robot_radius: 0.60
      track_unknown_space: true
      plugins: ["static_layer", "inflation_layer"]
      static_layer:
        plugin: "nav2_costmap_2d::StaticLayer"
        map_topic: /map
        map_subscribe_transient_local: True
        enabled: True
      inflation_layer:
        plugin: "nav2_costmap_2d::InflationLayer"
        enabled: True
        inflation_radius: 5.0
        cost_scaling_factor: 1.0
        inflate_unknown: false
        inflate_around_unknown: false
      always_send_full_costmap: True
